Public/Src/FrontEnd/MsBuild.Serialization/ProjectGraphWithPredictionsResult.cs

```diff
@@ -48,39 +49,46 @@ namespace BuildXL.FrontEnd.MsBuild.Serialization
         /// <nodoc/>
         public bool Succeeded { get; }
 
+        /// <summary>
+        /// A sorted list of the names of environment variables that could affect the build.

```

Comment:
>sorted list [](start = 14, length = 11)

why the fact that they are sorted is important?

Reply:
@smera, I didn't want to assume that subsequent builds of a project would always emit log events in the same order (or that they're received in the same order). So for consistency, I was returning them sorted. If the consumer (BuildXL) doesn't care, there's no need for it.

Reply:
There shouldn't be a need for that. Each variable is just registered as 'tracked' so graph construction caching is aware of it.

---
In reply to: [306449982](https://github.com/microsoft/BuildXL/pull/648#discussion_r306449982) [](ancestors = 306449982)

Reply:
The sorting will be removed in the next commit.